.travis.yml

```diff
@@ -57,8 +57,7 @@ jobs:
       deploy:
         provider: releases
         skip_cleanup: true
-        api_key:
-          secure: gtGyAu3m2v26KtkzQu4ljkLGLGKpCJAak0cO00zlvDvEGxwqmiH7yytG8uoAOise8Fp/1pF5YXrhpw6I7+WN1s8k+GJyCKoLG60wf9ODF4BR+9vGiwOfObs91yAfKz4oA/zzGG2eLdD59Y4VoRgY46OzsmGDbw9Zf34r+xGQ3fdrQCYbLItJzLwX/GEfWGBtVmJVb+BIuFrZ0UF1zTswC8+TkPDj/u+Pnc5bMXUqH7gUz2wrVdDc873EJjILB6U8iMIrFrKshr1CQ/C4JTJh7l6xiCJ5NS8Ik40jYrVfytMHo9cC93VkUnl1fNJuBo7kRQAbek1Jp7gmsfX1BQzmXp6p+VpgQmealqDWa+b25VhBS7T0Bz3nZvktAdRE6kbpVYgbS/YpDEQWNLaMreezd9taYjDFz+k3PlNCsMqPGfyU4jV1VViHuyFJvBuIt3JlHOm3mF/bNIdfDLVtkugl/etBoyITD0Gths+xNh997ctB9diGhlwmu3VkMEo173aEtYjpNPLQrkAFtA0476xLIJroPb4R2KlO31iC5tpUU9Ge/65ZUPaMaDIgR8kcLGmKvAdNwdDonZguT8Q/XlEPpn0MCmKbZcX2/G3l3+Sc2YN7EHeH1t7gGbQAjzHzsQCHLCCbL2p8X/AJzf6vQJRI1kEEs9QFwLg7vBjjT2n6Nzg=
+        api_key: $GH_TOKEN
```

Comment:
I recall having issues reading an env variable here, but maybe that was in an older travis enterprise version. Unfortunately it is hard to test this without actually submitting a release.

Reply:
Looks like the stacks repo is using a variable here, so I'm willing to give it a go!

Reply:
Yeah the stacks and the website repo use env variables so i think its okay